shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -949,13 +950,18 @@ public void detachFromFlutterEngine() {
     flutterEngine = null;
   }
 
+  @VisibleForTesting
```

Comment:
Are the tests in the same package (can this be package private)?

Reply:
different packages unfortunately.`io.flutter.embedding.android` vs `io.flutter.plugin.platform`